app/modules/individuals/resources.py

```diff
@@ -364,3 +364,121 @@ def get(self, individual):
         if individual is not None:
             others = individual.get_cooccurring_individuals()
             return others
+
+
+@api.route('/<uuid:individual_guid>/merge')
+@api.response(
+    code=HTTPStatus.NOT_FOUND,
+    description='Individual not found.',
+)
+@api.resolve_object_by_model(Individual, 'individual')
+class IndividualByIDMerge(Resource):
+    """
+    Merge from a list other Individual(s)
+    """
+
+    # permission on target individual is not needed, as user may only have access to other from_individuals
```

Comment:
You could use the standard way of handling permissions by creating a 'merge' action which would be more consistent than this.

Reply:
the permissions are pretty weird, as i try to outline below (and reference google doc).  basically for the N individuals (and their M >= N encounters), the _current user_ can meet one of 3 criteria: (a) has edit-priv on _all_ encounters; (b) has edit-priv on > 0 < M encounter; (c) has edit-priv on 0 encounters

the last case is rejected.  the first two cases have access to this endpoint, but with varying results: (a) gets to do the merge immediately; (b) gets to do a "merge request".

rather than handle this in a decorator, i chose to just let it pass through and handle the cases in the code, as i would be needing to make the distinction between (a) and (b) in there anyway.